.github/workflows/ci.yaml

```
@@ -210,7 +210,37 @@ jobs:
           echo "${VERSION}"
           make build-${{ matrix.dist }}
 
-  ### e2e tests ###
+  build-gpu-operator-bundle-image:
+    if: github.ref == 'refs/heads/main' # Runs only if the event is from a main branch commit
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        name: Check out code
+      - name: Calculate build vars
+        id: vars
+        run: |
+          echo "BRANCH_NAME=${GITHUB_REF_NAME}" >> $GITHUB_ENV
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+        with:
+          image: tonistiigi/binfmt:master
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Build bundle-image
+        env:
+          BUNDLE_IMAGE: "ghcr.io/nvidia/gpu-operator/gpu-operator-bundle:${{ env.BRANCH_NAME }}-latest"
```

You need this field as well
```suggestion
          BUNDLE_IMAGE: "ghcr.io/nvidia/gpu-operator/gpu-operator-bundle:${{ env.BRANCH_NAME }}-latest"
          VERSION: ""
```